Documentation/Upgrade/ceph-upgrade.md

```diff
@@ -1,13 +1,7 @@
 ---
-title: Upgrades
-weight: 3800
-indent: true
+title: Rook Ceph Upgrades
```

Comment:
In the preview, Chrome shows me a bright red warning page about this being a deceptive page. Safari works fine though. :)

Reply:
Let's just call this Rook upgrades, and in a future PR we can factor out the Ceph Upgrades page.
```suggestion
title: Rook Upgrades
```